doc/version_history.rst

```diff
@@ -6,6 +6,10 @@
 Version History
 ===============
 
+v1.23.2
```

Comment:
When adding new scripts, please make it a new minor release instead of patch, e.g. v1.24.0